### DISCUSSION TITLE
json5 support with comments

### DISCUSSION THREAD

**Discussion by lmajano** [FasterXML/jackson-jr#134] [Q&A] [answered]
Does Jackson jr support json5 with comments?

**Comment by cowtowncoder** [comment]
Jackson-jr supports whatever `jackson-core` supports, which only includes Java (/* ... */ and //) and Script ('#') comments; both optionally if enabled.

JSON5 is not supported (if I recall what that is), bits and pieces may be supported.

**Comment by lmajano** [comment]
Thanks so much! I did not know this.  Do you have a snippet on how to enable it in jackson jr?

**Reply by cowtowncoder** [reply, marked as answer]
You'd first configure `JsonFactory` with settings (`JsonReadFeature`s), then `JSON` instance created with factory. So something like:

```
JsonFactory jf = JsonFactory.builder().with(JsonReadFeature.ALLOW_JAVA_COMMENTS).build();
JSON jr = JSON.builder(jf).build();
POJO value = jr.beanFrom(POJO.class, jsonString);
```
